### DISCUSSION TITLE
[Android] Password Box Content Reveal Icon Behavior

### DISCUSSION THREAD

**Discussion by Soap-141** [unoplatform/uno#11861] [Q&A] [answered]
What is the normal `PasswordBox` reveal icon behavior for Android?

I just noticed while investigating for something that might be a bug that the reveal icon in the `PasswordBox` disappear on focus if there is already content in it. But if it's empty, the reveal icon stay there and is usable. Is this the normal Android behavior or it's a bug?

Thanks.

**Comment by FrozDark** [comment]
It works the same as on WinUI as far as I know

**Comment by Soap-141** [comment, marked as answer]
After asking Uno directly I now know that it's a normal behavior (for security).
